scripts/ci/analyze-commits.js

```diff
@@ -16,15 +16,19 @@ var blocked_statements = [
 ];
 
 var blockedRegex = new RegExp('\\+.*(' + blocked_statements.join('|') + ').*$', 'mg');
+var commit_range = process.env.TRAVIS_COMMIT_RANGE || 'HEAD~1';
```

Comment:
Huh, sorry I didn't got that. 

`TRAVIS_COMMIT_RANGE` is the most perfect solution, because it checks the complete PR and nothing else. And `HEAD~1` is just a fallback.